R/MltplxObject.R

```diff
@@ -28,7 +28,8 @@
 #' @export
 new_MltplxObject = function(x, y, marks,slide_id, xrange = NULL, yrange = NULL,
                             ps = NULL, bw = NULL,
-                            dist_metric = NULL, .dist_metric_name = NULL){
+                            dist_metric = NULL, .dist_metric_name = NULL,
+                            window = NULL){
```

Comment:
documentation for window here. sorry again.

Reply:
Also fixed!